src/msg_recorder/replayer.cc

```diff
@@ -19,7 +19,33 @@ void MessageReplayer::SetSpeedupRate(double rate) {
     speed_up_rate_ = rate;
 }
 
+void MessageReplayer::SetPostStartCallback(std::function<void()>&& post_start) {
```

Comment:
I would just name the arg "hook" or whatever uniform, so that code looks the same everywhere.